tests/Bug.t.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.29;
+
+import {Test} from "forge-std/Test.sol";
+import {FROST} from "contracts/FROST.sol";
+
+contract BugTest is Test {
+    function setUp() public {}
+
+    function test_Bug() public view {
```

Comment:
Can the test names be more descriptive please?

Reply:
fa241c688eb9c0a0b7048487a935857dd35fc1fb